### DISCUSSION TITLE
Help to visualize Open-DMARC-Analyzer

### DISCUSSION THREAD

**Discussion by TheFrenchPlug** [userjack6880/Open-DMARC-Analyzer#99] [Q&A]
Hello everyone,

I'm in internship and I'm trying to setting up Open-DMARC-Analyzer on an Ubuntu server. I've setting up Open-Parser-Report and php-Whois and now I'm trying to vizualize the dashboard of Open-DMARC-Analyzer. Can someone explain me how to do that ? 
I've setting up a basic php server on which I can access from a client computer. I guess I need to edit the /var/www/html but I have poor knownledge of PHP.

I'm here to answer your questions and hope someone can help me.

Have nice day.

**Comment by userjack6880** [comment]
PHP help is a bit out of the scope of this project, but all you'll need to do is git-clone this project into your web directory (usually `/var/www/html`) and follow the instructions on configuration and setting up the dependencies. If you have all of the software setup correctly, it should just work.
